man/systemd-system.conf.xml

```diff
@@ -408,7 +408,24 @@
         <varname>LimitXXX=</varname> directives and they accept the same parameter syntax,
         see <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>
         for details. Note that these resource limits are only defaults
-        for units, they are not applied to the service manager process (i.e. PID 1) itself.</para></listitem>
+        for units, they are not applied to the service manager process (i.e. PID 1) itself.</para>
+
+        <para>Most of these settings are unset, which means the resource limits are inherited from the parent process.
```

Comment:
"inherited from the parent process" might be confusing to people, since PID 1 on baremetal doesn't really have a "parent process". maybe say:

"from the kernel — or if invoked in a container from the container manager"

or so?